cassini_v2_files/spacecraft_clock_fix.py

```
@@ -0,0 +1,157 @@
+            continue
+
+        stop_count = stop_count_elem.text
+        exposure_duration = float(exposure_duration_elem.text) / 1000
+
+        # Convert stop count to UTC
+        stop_count_e = cspyce.scs2e(cassini_id, stop_count)
+        stop_count_utc = cspyce.et2utc(stop_count_e, 'ISOC', 3)
+
+        # Convert to datetime object, and subtract seconds to create
+        # spacecraft_clock_start_count in UTC
+        stop_count_dt = datetime.strptime(stop_count_utc, "%Y-%m-%dT%H:%M:%S.%f")
```

Ruff (0.15.6)

[warning] Naive datetime constructed using `datetime.datetime.strptime()` without %z

(DTZ007)